pom.xml

```diff
@@ -92,6 +112,63 @@
 					</systemProperties>
 				</configuration>
 			</plugin>
+
+			<plugin>
+				<groupId>org.apache.maven.plugins</groupId>
+				<artifactId>maven-surefire-plugin</artifactId>
+				<version>2.18.1</version>
+				<configuration>
+					<testFailureIgnore>true</testFailureIgnore>
```

Comment:
Because in a different way we will get an error
http://automated-testing.info/t/kak-izmenit-pom-chtoby-integrirovat-allure-sjunit-i-maven/9560